icons/sprint.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Sprint icon</title><path d="M2.13 23.995c5.493-1.808 13.457-6.02 18.996-9.782.465-.314.827-.607.827-1.118 0-.889-1.655-2.08-2.458-2.555-4.359 4.456-13.033 11.134-17.366 13.455M7.225 20.21c3.388-2.304 8.31-6.638 9.975-8.996.393-.554.683-.876.683-1.456 0-.952-1.128-1.65-1.94-2.135-1.946 4.51-6.316 10.125-8.718 12.587M9.816 16.682c2.344-2.967 4.188-6.454 4.188-8.958 0-1.308-.953-2.222-2.196-2.916.024 2.721-.934 9.101-1.992 11.874M10.312 13.82c.438-2.847.18-6.285-.379-8.442-.304-1.174-1.38-2.5-2.376-2.913 1.305 3.643 2.426 8.832 2.755 11.355M9.347 9.825C8.321 7.097 5.763 1.41 2 0c2.45 2.66 6.083 7.935 7.347 9.825"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Sprint icon</title><path d="M2.1514 24c5.4942-1.8084 13.4599-6.0213 19-9.784.4651-.3141.8272-.6072.8272-1.1183 0-.8892-1.6554-2.0804-2.4585-2.5555C15.16 14.9992 6.4844 21.6785 2.1504 24m5.0971-3.7858c3.3887-2.3045 8.3117-6.6394 9.977-8.9979.3932-.554.6832-.8761.6832-1.4563 0-.9522-1.1282-1.6503-1.9404-2.1354-1.9464 4.511-6.3173 10.1271-8.7198 12.5896m2.5915-3.5287c2.3445-2.9676 4.189-6.4554 4.189-8.9599 0-1.3083-.9533-2.2225-2.1965-2.9166.024 2.7216-.9342 9.1029-1.9925 11.8765m.4962-2.8626c.438-2.8476.18-6.2863-.3791-8.4438-.304-1.1742-1.3803-2.5005-2.3765-2.9136 1.3053 3.6438 2.4265 8.8339 2.7556 11.3574M9.37 9.827C8.3437 7.0985 5.7852 1.4103 2.0214 0 4.472 2.6606 8.1057 7.9367 9.37 9.827"/></svg>
```

Comment:
There seem to be some differences between PR (red) and [source](https://www.sprint.com/content/dam/sprint/us/en/design-elements/images/t-mobile-sprint-logo.svg) (black) - could you rebuild using the 'latest' logo?

![image](https://user-images.githubusercontent.com/1830867/103176613-1c64ca00-486b-11eb-926a-55def62b69d6.png)

Reply:
@adamrusted Fixed 👍🏼